### PR TITLE
add multilabel classification to validation

### PR DESCRIPTION
**Pull request by Tyrest**

### CODE REVIEW

File: src/autolabel/data_loaders/validation.py [hunk 1]
```diff
@@ -109,11 +109,41 @@ def validate(self, value: str):
...
+    def validate(self, value: str):
+        if value.startswith("[") and value.endswith("]"):
+            try:
+                seed_labels = eval(value)
+                if not isinstance(seed_labels, list):
+                    raise
+                unmatched_label = set(seed_labels) - self.labels_set
+                if len(unmatched_label) != 0:
+                    raise ValueError(
+                        f"labels: '{unmatched_label}' not in promt/labels provided in config "
```
Comment (rajasbansal) [inline src/autolabel/data_loaders/validation.py:132]: nit typo

File: src/autolabel/data_loaders/validation.py [hunk 2]
```diff
@@ -109,11 +109,41 @@ def validate(self, value: str):
...
+
+    label_column: str
+    labels_set: set  # A Multilabel Classification Task should have a unique set of labels in config
+
+    def validate(self, value: str):
+        if value.startswith("[") and value.endswith("]"):
+            try:
+                seed_labels = eval(value)
+                if not isinstance(seed_labels, list):
+                    raise
```
Comment (rajasbansal) [inline src/autolabel/data_loaders/validation.py:128]: trying to understand how the empty raise works - from what i read it means that it raises the last exception that was handled. Given we are not in an except clause here, what would an empty raise mean here?
I would suggest raising a ValueError just for readability

Comment (Tyrest) [reply src/autolabel/data_loaders/validation.py:128]: I was just copying the other validation functions but it makes more sense to raise a `ValueError` here so I'll add that now.

**Review by rajasbansal — APPROVED** [review]
lgtm

**Pull request merged**